package.json

```diff
@@ -24,9 +24,9 @@
   },
   "scripts": {
     "start": "cd public && http-server -p $PORT -i=false -c -1 --cors && cd -",
-    "postinstall": "POLAR_SEA_VIDEOS_URL='http://mozvr.com/projects/polarsea/content/videos/' npm run syncmedia && npm run syncmozvr",
-    "syncmedia": "if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.mp4' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.mp4 $POLAR_SEA_VIDEOS_URL'PolarSeaTrailer-3.mp4' ); fi; if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.webm' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.webm $POLAR_SEA_VIDEOS_URL'PolarSeaTrailer-3.webm' ); fi",
+    "postinstall": "npm run syncmedia && npm run syncmozvr",
+    "syncmedia": "if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.mp4' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.mp4 'http://mozvr.com/projects/polarsea/content/videos/PolarSeaTrailer-3.mp4' ); fi; if ! [ -f 'public/demos/polarsea/videos/PolarSeaTrailer-3.webm' ]; then ( mkdir -p public/demos/polarsea/videos/ && curl -o public/demos/polarsea/videos/PolarSeaTrailer-3.webm 'http://mozvr.com/projects/polarsea/content/videos/PolarSeaTrailer-3.webm' ); fi",
     "syncmozvr": "mkdir -p public/demos/mozvr-site/ && cd node_modules/mozvr-site/ && sed -e 's/\"url\":.*$/\"url\": \"\\/demos\\/mozvr-site\",/g' config.json > config_local.json && wintersmith build -o ../../public/demos/mozvr-site/ --config config_local.json && rm -f config_local.json && cd -",
-    "release": "release-it"
+    "release": "rm -rfv public/demos/polarsea/videos/* && release-it"
```

Comment:
should probably not remove the videos every time we release. should probably use a `.npmignore` instead.